tools/ts_library_builder/main_test.ts

```diff
@@ -0,0 +1,6 @@
+// Run this manually with:
+//
+//  ./node_modules/.bin/ts-node --project tools/ts_library_builder/tsconfig.json tools/ts_library_builder/main_test.ts
+
+import "./ast_util_test";
+import "./build_library_test";
```

Comment:
Rename `main_test.ts` to `test.ts` 

This isn't hooked into `tools/test.py` yet.  Add the following to `tools/unit_tests.py`, and check that `tools/test.py` fails if you change an assert inside one of your tests:

```python
    run([
        "node",
        "./node_modules/.bin/ts-node",
        "--project",
        "tools/ts_library_builder/tsconfig.json",
        "tools/ts_library_builder/main_test.ts"
    ])
```

Reply:
FYI I'd like to land https://github.com/denoland/deno/pull/958 before this, so that we can also get the benefit of output checking. 
This will change this slightly.